### PR TITLE
Implement form validation to check if the email format is in lowerCase

### PR DESCRIPTION
**Pull request by Estete9**
Added a small element after the submit button in the HTML
Added an event listener to the form
Checked the content of the email to check if its lower case
Added error message for when email format is incorrect

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @Estete9 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that does not have equal commits from the contributors. This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#work-and-submission-mode). and as such each person must commit to the project before submission

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -155,3 +155,26 @@ for (let i = 0; i < projectBtns.length; i += 1) {
...
+  const emailForm = form.elements['user-email'];
+  let emailValid = false;
+  if (emailForm.value === emailForm.value.toLowerCase()) {
+    emailValid = true;
+  }
+  if (emailValid) {
+    form.submit();
+  }
+  showError();
+});
```
Comment (algerina) [inline script.js:180]: - [ ] When using an email with uppercase letters the form is submitted normally and no error message is displayed, kindly fix that issue.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @Estete9 ,

Great job so far! 

May I highlight the following: 
- [x] Your project is professionally documented👏  ✔️
- [x] Your pull request is descriptive👏  ✔️



## CHANGES REQUIRED:muscle::clap:
Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: script.js (hunk 1 above)
Comment (Estete9) [reply script.js:180]: Hello @algerina! 
Can you explain it a bit more? I just tested it and the error message does show
image

you can see it, orange and next to the submit button

**Review by usorfaitheloho — APPROVED** [review]
Hi @Estete9  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**